### PR TITLE
Added description to all values of the SellerCloudPackageType enum.

### PR DESCRIPTION
**Pull request by DimitarChobanovSellerCloud**

### CODE REVIEW

**Review by StanislavChankovSellerCloud — APPROVED** [review]
Thanks @DimitarChobanovSellerCloud

File: src/SellerCloud.Core/Enums/PackageType/PackageType.cs [hunk 1]
```diff
@@ -1,234 +1,293 @@
...
         UspsFlatRateEnvelope = 106,
         /// <summary>
         /// USPS package code: FlatRateLegalEnvelope
         /// </summary>
+        [Description("Usps Flat Rate Legal Envelope")]
         UspsFlatRateLegalEnvelope = 107,
         /// <summary>
         /// USPS package code: FlatRatePaddedEnvelope
         /// </summary>
+        [Description("Usps Flat Rate Padded Envelope")]
```
Comment (stanimirsellercloud) [inline src/SellerCloud.Core/Enums/PackageType/PackageType.cs:53]: Please make sure the description used here matches what ShipBridge/CWA use. @MaxPower-tsvetkov-k can help if needed.

For example, in SB/CWA (and regular branded material), `USPS` is spelt all-uppercase.

File: src/SellerCloud.Core/Enums/PackageType/PackageType.cs [hunk 2]
```diff
@@ -1,234 +1,293 @@
...
         UspsSmallFlatRateEnvelope = 112,
         /// <summary>
         /// USPS package code: SmallFlatRateBox
         /// </summary>
+        [Description("Usps Small Flat Rate Box")]
         UspsSmallFlatRateBox = 113,
         /// <summary>
         /// USPS package code: MediumFlatRateBox
         /// </summary>
+        [Description("Usps Medium Flat Rate Box")]
```
Comment (stanimirsellercloud) [inline src/SellerCloud.Core/Enums/PackageType/PackageType.cs:83]: I think the flat rate boxes do not have the `USPS` prefix. Please check in ShipBridge. It is possible that other packaging types have slightly different naming also.

File: src/SellerCloud.Core/Enums/PackageType/PackageType.cs [hunk 3]
```diff
@@ -1,234 +1,293 @@
...
         /// <summary>
         /// USPS package code: Package
         /// </summary>
+        [Description("Usps Package")]
         UspsPackage = 121,
 
         /// <summary>
         /// FedEx package code: YOUR_PACKAGING
         /// </summary>
+        [Description("FedEx Your Packaging")]
```
Comment (stanimirsellercloud) [inline src/SellerCloud.Core/Enums/PackageType/PackageType.cs:124]: Similarly, `Your Packaging` is used without the `FedEx` prefix.

**Review by stanimirsellercloud — CHANGES_REQUESTED** [review]
We should ensure that all the packaging types defined in `Core` have corresponding values as expected by ShipBridge and CWA.

These string values are set by plugins also, so they must match exactly (casing, hyphens, with/without prefixes where needed).

**Review by zdravko-g — APPROVED** [review]

File: src/SellerCloud.Core/Enums/PackageType/PackageType.cs (hunk 3 above)
Comment (DimitarChobanovSellerCloud) [reply src/SellerCloud.Core/Enums/PackageType/PackageType.cs]: done

File: src/SellerCloud.Core/Enums/PackageType/PackageType.cs (hunk 2 above)
Comment (DimitarChobanovSellerCloud) [reply src/SellerCloud.Core/Enums/PackageType/PackageType.cs]: checked all names and made changes

File: src/SellerCloud.Core/Enums/PackageType/PackageType.cs (hunk 1 above)
Comment (DimitarChobanovSellerCloud) [reply src/SellerCloud.Core/Enums/PackageType/PackageType.cs]: checked all names and made changes

File: src/SellerCloud.Core/Enums/PackageType/PackageType.cs [hunk 4]
```diff
@@ -1,234 +1,293 @@
...
 
         /// <summary>
         /// UPS package code: 01
         /// </summary>
+        [Description("UPS Letter")]
         UpsLetter = 300,
         /// <summary>
         /// UPS package code: 02
         /// </summary>
+        [Description(@"Package/customer supplied")]
```
Comment (stanimirsellercloud) [inline src/SellerCloud.Core/Enums/PackageType/PackageType.cs:185]: All other entries look great but this one I do not recognise. I think we should change it to `Package`.

**Pull request merged**